packages/runtime/runtime-definitions/src/id-compressor/persisted-types/0.0.1.ts

```diff
@@ -6,21 +6,30 @@
 import type { SessionId } from "../identifiers";
 
 /**
- * The serialized contents of an IdCompressor, suitable for persistence in a summary.
+ * The serialized contents of an {@link @fluidframework/container-runtime#IdCompressor}, suitable for persistence
```

Comment:
Runtime definitions has no dep on container-runtime (nor should it, I think). Will the links in this file work? Maybe these should link to `IIdCompressor` in this package instead of the implementation.

Reply:
Ah, good call. Yes, these should link to the interface. Will fix.

Reply:
And links to packages not depended on _are_ allowed by the tooling, but I would generally consider it an anti-pattern.